mautic/mautic
# [UI] Minor improvements for panels

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR:
- Adds support for dark theme for panels
- Adds a smooth transition for panel borders
- Makes panel footers transparent
- Removes borders from page headers
- Removes a duplicated code from form.css

![image](https://github.com/mautic/mautic/assets/116097999/72e99e75-255d-4a57-917a-c7adc8fb722b)

![image](https://github.com/mautic/mautic/assets/116097999/178977bf-4684-4563-be99-274cd41bae06)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts
3. Create new
4. Save & Close
5. Hover over their details
